src/switch.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python3
+#
+# igcollect - Switch stats:
+#             - port traffic
+#             - port errors
+#             - cpu usage
+#
+# Copyright (c) 2017, InnoGames GmbH
+#
+
+from argparse import ArgumentParser
+from time import time
+from pysnmp import proto, error
+from pysnmp.hlapi import (
+        CommunityData,
+        ContextData,
+        getCmd,
+        bulkCmd,
+        ObjectType,
+        ObjectIdentity,
+        SnmpEngine,
+        UdpTransportTarget,
+        UsmUserData,
+        usmHMACSHAAuthProtocol,
+        usmDESPrivProtocol,
+    )
+
+import re
+import sys
+
+snmp_engine = SnmpEngine()
+context_data = ContextData()
+
+
+OIDs = {
```

Comment:
I made them all upper case, although is is for plural and you know that I don't agree with that notation.